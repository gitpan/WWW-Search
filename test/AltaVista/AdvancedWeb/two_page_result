http://www.altavista.digital.com/cgi-bin/query?what=web&text=yes&fmt=d&pg=aq&q=LSAM+and+ISI+and+work
/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=10
/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=20
/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=30
