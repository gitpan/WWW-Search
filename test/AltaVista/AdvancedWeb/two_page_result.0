<html><head>
<title>AltaVista: Advanced Query LSAM and ISI and IB</title>
</head><body>
<font size=+1>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font> - 
<a href="/cgi-bin/query?text">Search</a> - 
<a href="/av/content/zones.html">Zones</a> - 
<a href="/av/content/services.html">Services</a> - 
<a href="/av/content/help.htm">Help</a> - 
<a href="/av/content/questions.htm">Feedback</a>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=aq>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><B>Search the Web for documents in</B> <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><font size=-1><br><input type=text name=r size=50 value="">
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit><br>
 Boolean expression:<br>
<textarea name=q cols=50 rows=3 maxlength=800 wrap=virtual>LSAM and ISI and IB</textarea><br>Range of Dates (e.g. 21/Mar/96): <b> From </b><INPUT TYPE=text NAME=d0 SIZE=9 MAXLENGTH=32 VALUE="">
<b> To </b><INPUT TYPE=text NAME=d1 SIZE=9 MAXLENGTH=32 VALUE=""><br><INPUT TYPE=checkbox name=fmt value=n> <b>Count documents matching the boolean expression.</b> 
<P><font size=-2><a href="/cgi-bin/query?pg=q&text=yes">Search</a> - Advanced - <a href="http://altavista.switchboard.com/bin/cgiqa.dll?MG=&MEM=212">Find a Person</a> - <a href="http://altavista.switchboard.com/bin/cgidir.dll?MG=&MEM=212">Find a Business</a> - <a href="http://altavista.looksmart.com/altavista.html">Browse Categories</a> - <a href="/cgi-bin/query?pg=aq&what=news&text=yes">Search Usenet</a></font><BR></font></FORM>
<font size=-1>About <b>16</b> matches were found. </font><P><font size=-1>Pages: <b>1</b> <a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=10&c9k">2</a>
<a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=10&c9k">[<b>&gt;&gt;</b>]</a> <P>
</font><font size=-1><dl>
<dt><b>1. </b><a href="http://www.isi.edu/div7/ib/example.html"><b>IB Research Plan</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/ib/example.html]</font><br>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><font size=-2>Last modified 13-Jan-97 - page size 6K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fib%2fexample%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>2. </b><a href="http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/test/Lycos/README.html"><b>/usr/local/lib/perl5/site_perl/WAIT/DB/cpan/data/WWW-Search/test/Lycos/README</b></a><dd>
<font size=-2>[<b>URL:</b> theory.uwinnipeg.ca/CPAN/data/WWW-Search/test/Lycos/README.h]</font><br>
Contained in WWW-Search-1.014.tar.gz. requires perl 5.002 or later. uses LWP::UserAgent in libwww-perl-5.30.tar.gz. NAME. DESCRIPTION. NAME. WWW-Search...<br><font size=-2>Last modified 27-Mar-98 - page size 1K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2ftheory%2euwinnipeg%2eca%2fCPAN%2fdata%2fWWW%2dSearch%2ftest%2fLycos%2fREADME%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>3. </b><a href="http://www.isi.edu/~oswal/design.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/~oswal/design.html]</font><br>
IB - Reliable Multicast Transport protocol. Reliability Mechanism: The idea of multicasting NACks is based on the fact that multicast losses are...<br><font size=-2>Last modified 22-Oct-97 - page size 4K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7eoswal%2fdesign%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>4. </b><a href="http://www.infospheres.caltech.edu/mailing_lists/dist-obj/msg00886.html"><b>[Fwd: LSAM: Large-Scale Active Middleware (formerly Cities Online)]</b></a><dd>
<font size=-2>[<b>URL:</b> www.infospheres.caltech.edu/mailing_lists/dist-obj/msg00886.]</font><br>
Date Prev][Date Next][Thread Prev][Thread Next][Date Index][Thread Index] [Fwd: LSAM: Large-Scale Active Middleware (formerly Cities Online)] To:...<br><font size=-2>Last modified 9-Apr-98 - page size 11K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2einfospheres%2ecaltech%2eedu%2fmailing%5flists%2fdist%2dobj%2fmsg00886%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>5. </b><a href="http://www.isi.edu/div7/ib/discussion.html"><b>IB Discussion</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/ib/discussion.html]</font><br>
IB Discussion. Response time and bandwidth are key resources in this project. Current Internet user interfaces, such as Web browsers, Gopher and WAIS...<br><font size=-2>Last modified 13-Jan-97 - page size 6K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fib%2fdiscussion%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>6. </b><a href="http://www.isi.edu/div7/cities/ib/projects.html"><b>IB Research Plan</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/cities/ib/projects.html]</font><br>
IB Research Projects. &nbsp; &nbsp; Lowlat. &nbsp;Anticipatory document caching reduces request latency. &nbsp; Multicast Web Tuner. Multicasting popular..<br><font size=-2>Last modified 12-Feb-98 - page size 2K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fib%2fprojects%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>7. </b><a href="http://www.isi.edu/div7/ib/plan.html"><b>IB Research Plan</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/div7/ib/plan.html]</font><br>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><font size=-2>Last modified 13-Jan-97 - page size 4K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fib%2fplan%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>8. </b><a href="http://www.isi.edu/~oswal/req.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/~oswal/req.html]</font><br>
IB - Reliable Multicast Transport protocol. &nbsp Reliability Mechanism. Retransmission is event driven (off-line). Retransmission requests uses selactive.<br><font size=-2>Last modified 22-Oct-97 - page size 2K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7eoswal%2freq%2ehtml&language=en">Translate</a> ]</font><P>
<dt><b>9. </b><a href="http://www.isi.edu/lsam/about/"><b>About LSAM</b></a><dd>
<font size=-2>[<b>URL:</b> www.isi.edu/lsam/about/]</font><br>
About LSAM. What is LSAM. LSAM is developing middleware infrastructure to support scalable distributed information services. This is embodied in the LSAM..<br><font size=-2>Last modified 26-Mar-98 - page size 9K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2flsam%2fabout%2f&language=en">Translate</a> ]</font><P>
<dt><b>10. </b><a href="http://xent.ics.uci.edu/FoRK-archive/jan98/0429.html"><b>FoRK Archive: LSAM: Large-Scale Active Middleware (formerly Cities Online)</b></a><dd>
<font size=-2>[<b>URL:</b> xent.ics.uci.edu/FoRK-archive/jan98/0429.html]</font><br>
LSAM: Large-Scale Active Middleware (formerly Cities Online) Rohit Khare (rohit@uci.edu) Tue, 03 Feb 1998 00:05:42 -0800. Messages sorted by: [ date ][...<br><font size=-2>Last modified 3-Feb-98 - page size 9K - in English [ <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fxent%2eics%2euci%2eedu%2fFoRK%2darchive%2fjan98%2f0429%2ehtml&language=en">Translate</a> ]</font><P>
</dl>
</font><font size=-1>Pages: <b>1</b> <a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=10&c9k">2</a>
<a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=10&c9k">[<b>&gt;&gt;</b>]</a> <P>
</font><font size=-2></font><P>
<br>
<font size=-2 face=arial>
<a href="/cgi-bin/query?text">Search</a> | 
<a href="/av/content/zones.html">Zones</a> | 
<a href="/av/content/services.html">Services</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright (c) 1995-98 
<a href="http://www.digital.com">Digital Equipment Corporation</a> | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
