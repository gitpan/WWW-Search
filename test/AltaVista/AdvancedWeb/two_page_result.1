<html><head>
<title>AltaVista: Advanced Query LSAM and ISI and IB</title>
</head><body>
<font size=+1 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=aq>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><B>Enter ranking keywords in</B>
<SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><font size=-1><br><input type=text name=r size=50 value="">
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit><br>
 Enter boolean expression:<br>
<textarea name=q cols=50 rows=3 maxlength=800 wrap=virtual>LSAM and ISI and IB</textarea><br>Range of Dates (e.g. 21/Mar/96): <b> From </b><INPUT TYPE=text NAME=d0 SIZE=9 MAXLENGTH=32 VALUE="">
<b> To </b><INPUT TYPE=text NAME=d1 SIZE=9 MAXLENGTH=32 VALUE=""><br><INPUT TYPE=checkbox name=fmt value=n> <b>Count documents matching the boolean expression.</b> 
<P><font size=-1><a href="/cgi-bin/query?pg=q&what=web&text=yes">Search</a> - Advanced - <a href="/cgi-bin/query?pg=aq&what=news&text=yes">Usenet</a></font><BR></font></FORM>
<font size=-1><b>AltaVista found about 13 Web pages for you. </font></b><P><font size=-1><b>Result Pages: </b><a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=0&c9k">[<b>&lt;&lt; Prev</b>]</a>  &nbsp;<a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=0&c9k">1</a>
 <b>2</b><br>
</font><font size=-1><dl><dt><b>11. </b><a href="http://www.isi.edu/~lsam/ib/rmc/design.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Reliability Mechanism: The idea of multicasting NACks is based on the fact that multicast losses are...<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/design.html<br>
Last modified 12-Feb-98 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2fdesign%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>12. </b><a href="http://www.isi.edu/~lsam/ib/rmc/links.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Here are a few good links for Multicast Transport Protocols. ...<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/links.html<br>
Last modified 12-Feb-98 - page size 1K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2flinks%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>13. </b><a href="http://www.isi.edu/~lsam/ib/rmc/req.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol.  Reliability Mechanism. Retransmission is event driven (off-line). Retransmission requests uses selactive...<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/req.html<br>
Last modified 12-Feb-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2freq%2ehtml&language=en">Translate</a>&nbsp;]</dl>
</font><font size=-1><b>Result Pages: </b><a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=0&c9k">[<b>&lt;&lt; Prev</b>]</a>  &nbsp;<a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+IB&stq=0&c9k">1</a>
 <b>2</b><br>
</font><font size=-2></font><P>
<br>
<font size=-2 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Home</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
