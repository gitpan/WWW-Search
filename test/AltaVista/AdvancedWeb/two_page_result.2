<html><head>
<title>AltaVista: Advanced Query LSAM and ISI and work</title>
</head><body>
<a href="/cgi-bin/query?text=yes"><b>AltaVista Search</b></a><br>

<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=aq>
<INPUT TYPE=hidden NAME=stq VALUE=20>
<INPUT TYPE=hidden NAME=text VALUE=yes>

<a href="http://babelfish.altavista.digital.com/cgi-bin/translate?">Translations</a> .
<a href="http://altavista.iname.com/">Free eMail</a> .
<a href="http://altavista.looksmart.com/altavista.html">Browse by Subject</a> . 
<a href="http://altavista.switchboard.com/bin/cgiqa.dll?MG=&MEM=212">People Search</a> . 
<a href="http://altavista.switchboard.com/bin/cgidir.dll?MG=&MEM=212">Business Search</a>
<br>
<font size=-1 face=arial><B>Search <SELECT NAME=what><OPTION VALUE=web SELECTED>the Web<OPTION VALUE=news >Usenet</SELECT>
for documents in <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh >Chinese
<OPTION VALUE=cs >Czech
<OPTION VALUE=da >Danish
<OPTION VALUE=nl >Dutch
<OPTION VALUE=en >English
<OPTION VALUE=et >Estonian
<OPTION VALUE=fi >Finnish
<OPTION VALUE=fr >French
<OPTION VALUE=de >German
<OPTION VALUE=el >Greek
<OPTION VALUE=he >Hebrew
<OPTION VALUE=hu >Hungarian
<OPTION VALUE=is >Icelandic
<OPTION VALUE=it >Italian
<OPTION VALUE=ja >Japanese
<OPTION VALUE=ko >Korean
<OPTION VALUE=lv >Latvian
<OPTION VALUE=lt >Lithuanian
<OPTION VALUE=no >Norwegian
<OPTION VALUE=pl >Polish
<OPTION VALUE=pt >Portuguese
<OPTION VALUE=ro >Romanian
<OPTION VALUE=ru >Russian
<OPTION VALUE=es >Spanish
<OPTION VALUE=sv >Swedish
</SELECT></B></font><br><textarea name=q cols=65 rows=3 wrap=virtual>LSAM and ISI and work</textarea><br><a href="/av/content/help_advanced.htm#ranking"><b>Ranking:</b></a><input type=text name=r size=25 maxlength=800 value="">
<b><a href="/av/content/help_advanced.htm#dates" title="Keep documents published after this date"><font size=-1 face=arial><b> From:</b></font></a> <INPUT TYPE=text NAME=d0 SIZE=9 MAXLENGTH=32 VALUE="21/Mar/86">
<b><a href="/av/content/help_advanced.htm#dates" title="Keep documents published before this date"><font size=-1 face=arial><b> To:</b></font></a> <INPUT TYPE=text NAME=d1 SIZE=9 MAXLENGTH=32 VALUE=""><br><INPUT TYPE=checkbox name=fmt value=n> <b>Give me only a precise count of matches.</b> 
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit>
 <BR><a href="/av/content/help_advanced.htm">Help</a> . <a href="/cgi-bin/query?pg=tmpl&v=pref.html&text=yes">Preferences</a> . <a href="/cgi-bin/query?pg=aq&text=yes">New Search</a> . <a href="/cgi-bin/query?pg=q&text=yes">Simple Search</a><br>Click here to find related books at <a href="http://www.amazon.com/exec/obidos/av-search/q=LSAM+and+ISI+and+work">amazon.com</a>
<br>




About <b>40</b> documents match your query.<dl>
<dt><b>21. </b><a href="http://www.iihe.ac.be/rfc/rfc2140.txt"><b>No Title</b></a><dd>Network Working Group J. Touch Request for Comments: 2140 ISI Category: Informational April 1997 TCP Control Block Interdependence Status of this Memo...<br><i>http://www.iihe.ac.be/rfc/rfc2140.txt - size 25K - 25-Apr-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eiihe%2eac%2ebe%2frfc%2frfc2140%2etxt&language=en">Translate</a></i><br>
<P><dt><b>22. </b><a href="http://gost.isi.edu/people/johnh/SOFTWARE/"><b>Software Written by John Heidemann</b></a><dd>Software Written by John Heidemann. Writing software is both my job and one of my hobbies. The following packages are currently available: Operating...<br><i>http://gost.isi.edu/people/johnh/SOFTWARE/ - size 8K - 15-Aug-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2fpeople%2fjohnh%2fSOFTWARE%2f&language=en">Translate</a></i><br>
<P><dt><b>23. </b><a href="http://www.isi.edu/div7/cities/about/replication.html"><b>About LSAM Replication</b></a><dd>About LSAM Replication. Why Replication? Although the Web is widely used today, for it to be truly effective part of daily work, it must provide both...<br><i>http://www.isi.edu/div7/cities/about/replication.html - size 4K - 6-Jan-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fabout%2freplication%2ehtml&language=en">Translate</a></i><br>
<P><dt><b>24. </b><a href="http://gost.isi.edu/~johnh/WORK/"><b>John Heidemann's Work</b></a><dd>John Heidemann's Work. I am computer scientist at Information Sciences Institute and a research assistant professor at the University of Southern...<br><i>http://gost.isi.edu/~johnh/WORK/ - size 6K - 10-Feb-98 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2f%7ejohnh%2fWORK%2f&language=en">Translate</a></i><br>
<P><dt><b>25. </b><a href="http://gost.isi.edu/~johnh/SOFTWARE/"><b>Software Written by John Heidemann</b></a><dd>Software Written by John Heidemann. Writing software is both my job and one of my hobbies. The following packages are currently available: Operating...<br><i>http://gost.isi.edu/~johnh/SOFTWARE/ - size 8K - 12-Feb-98 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2f%7ejohnh%2fSOFTWARE%2f&language=en">Translate</a></i><br>
<P><dt><b>26. </b><a href="http://gost.isi.edu/brian/resume/work.html"><b>Brian Tung</b></a><dd>Brian Tung. Computer Scientist at ISI. I started working at ISI in November 1994. For the moment, I'm splitting my time between two projects: Security...<br><i>http://gost.isi.edu/brian/resume/work.html - size 1K - 17-Jun-96 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2fbrian%2fresume%2fwork%2ehtml&language=en">Translate</a></i><br>
<P><dt><b>27. </b><a href="http://www.isi.edu/div7/cities/publications/phttp_tcp_interactions/footnode.html"><b>Footnotes</b></a><dd>Footnotes<br><i>http://www.isi.edu/div7/cities/publications/phttp_tcp_interactions/footnode.html - size 2K - 27-Feb-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fpublications%2fphttp%5ftcp%5finteractions%2ffootnode%2ehtml&language=en">Translate</a></i><br>
<P><dt><b>28. </b><a href="http://gost.isi.edu/~johnh/CHANGES/"><b>Changes to John Heidemann's Web Pages</b></a><dd>Changes to John Heidemann's Web Pages. A list of changes to my web pages, in reverse chronological order: 9 February 1998. Updates about sound on the...<br><i>http://gost.isi.edu/~johnh/CHANGES/ - size 10K - 10-Feb-98 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2f%7ejohnh%2fCHANGES%2f&language=en">Translate</a></i><br>
<P><dt><b>29. </b><a href="http://www2.umin.u-tokyo.ac.jp/internet/drafts/draft-touch-tcp-interdep-01.txt"><b>No Title</b></a><dd>INTERNET-DRAFT Joe Touch draft-touch-tcp-interdep-01.txt ISI March 28, 1996 Expires: Sept. 28, 1996 TCP Control Block Interdependence Status of this Memo..<br><i>http://www2.umin.u-tokyo.ac.jp/internet/drafts/draft-touch-tcp-interdep-01.txt - size 26K - 18-Apr-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww2%2eumin%2eu%2dtokyo%2eac%2ejp%2finternet%2fdrafts%2fdraft%2dtouch%2dtcp%2dinterdep%2d01%2etxt&language=en">Translate</a></i><br>
<P><dt><b>30. </b><a href="http://www.ietf.org/proceedings/97apr/97apr-final/xrtftr20.htm"><b>xrtftr~1.htm</b></a><dd>Current Meeting Report. Slides. Attendees List. Large Scale Multicast Applications (lsma) NOTE: This charter is a snapshot of that in effect at the time...<br><i>http://www.ietf.org/proceedings/97apr/97apr-final/xrtftr20.htm - size 18K - 30-Jul-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eietf%2eorg%2fproceedings%2f97apr%2f97apr%2dfinal%2fxrtftr20%2ehtm&language=en">Translate</a></i><br>
<P></dl>
<font size=-1 face=arial><font size=-1></font>
</font><P><a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=10">[Prev]</a> <a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=0">1 </a><a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=10">2 </a>3 <a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=30">4 </a><a href="/cgi-bin/query?pg=aq&text=yes&q=LSAM+and+ISI+and+work&stq=30">[Next]</a><br>
<HR><font face="arial, helvetica" size=-1>
<a href="/av/content/av_network.htm">Our Network</a> | 
<a href="/av/content/addurl.htm">Add/Remove URL</a> | 
<a href="/av/content/questions.htm">Feedback</a> | 
<a href="/av/content/help.htm">Help</a><br>

<a href="http://www.doubleclick.net/jumps/jump1217.htm">Advertising Info</a> | 
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/av/content/jobs.htm">Jobs</a> | 
<a href="/cgi-bin/query?pg=q">Graphics</a></font><HR>
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy Statement</a><br>

<a href="http://www.digital.com"><b>DIGITAL</b></a> Copyright 1995-98. All Rights Reserved
</FORM>
</BODY></HTML>
