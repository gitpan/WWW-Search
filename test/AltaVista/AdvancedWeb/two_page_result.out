http://www.acm.rpi.edu/~albert/pilot/Jan97/0095.html
http://www.acm.rpi.edu/~albert/pilot/Jan97/0066.html
http://andrew2.andrew.cmu.edu/rfc/rfc2140.html
http://www2.umin.u-tokyo.ac.jp/internet/drafts/draft-touch-tcp-interdep-01.txt
http://www.isi.edu/~faber/
http://info.internet.isi.edu:70/0/in-drafts/files/draft-tung-transsec-sts-01.txt
http://www.wls.org/resume.html
http://boggy.epistat.m.u-tokyo.ac.jp:6080/=@=:www2.umin.u-tokyo.ac.jp/internet/drafts/1id-abstracts.txt
http://www.isi.edu/atomic2/lab.html
http://gost.isi.edu/brian/resume/work.html
http://www.isi.edu/~johnh/home_page.html
http://www.isi.edu/lsam/tools/WWW_SEARCH/
http://www.ics.uci.edu/pub/ietf/http/hypermail/1997q1/0155.html
http://gost.isi.edu/~johnh/SOFTWARE/APACHE/apache_1.0_isi_mmap_960524.patch
http://hplbwww.hpl.hp.com/people/ange/archives/archives-97/http-wg-archive/0151.html
http://gost.isi.edu/brian/cities/
http://www.w3.org/Protocols/HTTP/Issues/touch.html
http://gost.isi.edu/people/johnh/WORK/
http://hplbwww.hpl.hp.com/people/ange/archives/archives-97/http-wg-archive/0250.html
http://gost.isi.edu/~johnh/SOFTWARE/APACHE/
http://www.iihe.ac.be/rfc/rfc2140.txt
http://gost.isi.edu/people/johnh/SOFTWARE/
http://www.isi.edu/div7/cities/about/replication.html
http://gost.isi.edu/~johnh/WORK/
http://gost.isi.edu/~johnh/SOFTWARE/
http://gost.isi.edu/brian/resume/work.html
http://www.isi.edu/div7/cities/publications/phttp_tcp_interactions/footnode.html
http://gost.isi.edu/~johnh/CHANGES/
http://www2.umin.u-tokyo.ac.jp/internet/drafts/draft-touch-tcp-interdep-01.txt
http://www.ietf.org/proceedings/97apr/97apr-final/xrtftr20.htm
http://boggy.epistat.m.u-tokyo.ac.jp:6080/=@=:www2.umin.u-tokyo.ac.jp/internet/drafts/1id-abstracts.txt
http://www.cs.arizona.edu/xkernel/www/ipsec/ipsec-mail4.html
http://www.cs.arizona.edu/xkernel/www/ipsec/ipsec-mail3.html
http://www.dc.luth.se/doc/rfc/rfc2140.txt
http://www.isi.edu/lsam/people.html
http://hplbwww.hpl.hp.com/people/ange/archives/archives-97/http-wg-archive/0195.html
http://gigahertz.lerc.nasa.gov/~mallman/tcpsat-archive
http://www.moviola.com/movioladigital/communications/foundation.html
http://reference.nrcs.usda.gov/ietf/rfc/2200/rfc2140.htm
http://www.isi.edu/~johnh/SOFTWARE/APACHE/apache_1.0_isi_mmap_961202.patch
